physionet-django/project/models.py

```diff
@@ -419,16 +456,12 @@ class Metadata(models.Model):
     # Main body descriptive metadata
     title = models.CharField(max_length=200, validators=[validate_alphaplus])
     abstract = SafeHTMLField(max_length=10000, blank=True)
-    background = SafeHTMLField(blank=True)
-    methods = SafeHTMLField(blank=True)
-    content_description = SafeHTMLField(blank=True)
-    usage_notes = SafeHTMLField(blank=True)
-    installation = SafeHTMLField(blank=True)
-    acknowledgements = SafeHTMLField(blank=True)
-    conflicts_of_interest = SafeHTMLField(blank=True)
+    release_notes = SafeHTMLField(blank=True)
```

Comment:
Perhaps abstract and release_notes should be part of the SectionContent model? This would deal with the ordering issue that we have on the content form at the moment (release notes appearing after the abstract). It's possible that future projects will not have abstracts or release notes.